docs/index.html

```diff
@@ -426,10 +426,6 @@
         padding-top: 1rem !important;
         padding-bottom: 1rem !important;
       }
-      .markdown-section pre {
-        max-height: 40rem;
-        overflow-y: scroll;
-      }
```

Comment:
intentional?

Reply:
Yes, I added it when we talked about it, but you added the same thing on a different selector just below that one, creating two scrolls